routers/teams.js

```diff
@@ -144,4 +144,33 @@ router.post("/search/teamname", async (req, res) => {
   }
 });
 
+//based on the userid to get user's teams
+router.get("/getTeam/:userid", async (req, res) => {
+  try {
+    const { userid } = req.params;
+
+    const getAllTeam = await pool.query(
+      "SELECT * FROM tbl_team_players WHERE userid = $1",
+      [userid]
+    );
+
+    const teamIds = getAllTeam.rows.map((item) => item.teamid);
+
+    const teamInfo = [];
+
+    for (const teamid of teamIds) {
```

Comment:
Instead of this, can't you do a JOIN statement where you join the tbl_team and tbl_team_players by teamid? Rather than having to call our database multiple times for the team ID? Think it will be like this: 

```SELECT * FROM tbl_team t JOIN tbl_team_players tp ON t.teamid = tp.teamid WHERE tp.userid = 'f7a0ab13-1573-4716-9423-95d02b8d6732';``` 

Rather than doing a for loop, that sql statement returns all the team information that a user is a part of

Reply:
I think you are right
I will fix it while I got home from work.
Really nice advice brooooooooooo